### ISSUE TITLE
Do not auto-launch after installation without prompting user for where to connect to.

### ISSUE THREAD

**Issue by MicahZoltu** [parity-js/shell#181]
I'm trying to use this to connect to a remote node.  I expressly do not want parity installed/running on my machine due to network/drive space limitations.  When I run the installer, it immediately launches parity and begins syncing without giving me an opportunity to stop it and tell it where to connect to.

**Comment by Tbaut** [comment]
The idea behind this behaviour was to make beginner users launch a parity node in the background. If you wish to connect to your own server, we recommend using (SSH tunelling)[https://wiki.parity.io/Wallet-Remote-Access#recommended-ssh-tunneling] and the flags `--no-run-parity` to make sure no node is launched in the background.

You know all of this, but writing it for anybody else visiting this thread :)

**Comment by indigoram89** [comment]
I launch Parity UI on Mac OS (not from the terminal) and I have the same question?

**Comment by axelchalon** [comment]
@indigoram89 if you're running Parity with websockets on non-standard ports, currently the only way to use your running instance is to launch Parity UI from the terminal with the flags `--ws-interface=<YOUR CUSTOM IP> --ws-port=<YOUR CUSTOM PORT>`

**Comment by indigoram89** [comment]
@axelchalon I'm running Parity on standard ports, but I don't want to run local Parity node. I install Parity UI on Mac from parity-ui-0.3.2.pkg file and I don't understand how can I run It from the terminal?

**Comment by indigoram89** [comment]
@axelchalon Wow! Thank you very much for helping me!

If I connecting to Parity node with SSH tunelling, so Parity UI will connect to It without flags?

**Comment by indigoram89** [comment]
I did it with SSH tunelling - It's the great experience for me! Thank you!

**Comment by stone212** [comment]
@axelchalon Your suggestion is an excellent one.

**Comment by stone212** [comment]
@axelchalon you said 

> Also, currently we download Parity if the binary cannot be found on the computer (even if Parity UI can connect to Parity) so that we can run parity signer new-token programmatically.

Can you explain how this would work?  If you are connected to a server running Parity then you will run `parity signer new-token` on the *server* so there is no need to have the binary on the client machine.

**Comment by Tbaut** [comment]
> I can't stress enough that we can not run Parity on the client computer, period.

Please remain kind and polite. This software is provided for free, with support for free... We recommend to use other software as Parity UI is not maintained anymore. This software is open source, you are very welcome to make PR if you wish to make changes.

**Comment by axelchalon** [comment]
> Can you explain how this would work? If you are connected to a server running Parity then you will run parity signer new-token on the server so there is no need to have the binary on the client machine.

To streamline the user experience and so that the user doesn't have to run `parity signer new-token`, we launch this command automatically from Parity UI. For this to be possible, the binary needs to be accessible (located on the machine)

**Comment by stone212** [comment]
Hi @axelchalon you didn't actually answer.  Let me get more specific:

>  To streamline the user experience and so that the user doesn't have to run parity signer new-token, we launch this command automatically from Parity UI. For this to be possible, the binary needs to be accessible (located on the machine)

It sounds like you are saying that the *local* binary allows you to run "parity signer new-token" *on the server* through the UI.  This doesn't make sense to me.  So my question is "Can you explain how this would work?"  How is it that having Parity installed on my local computer allows me to run the `parity signer new-token` command on the server?

**Comment by stone212** [comment]
@Tbaut 

> Please remain kind and polite. 

Who is being unkind or impolite?  No one.  Not you, not me.  I explained a use case.  My use case (and that of anyone who lives somewhere that mining is frowned upon for example) is that I absolutely "can not run Parity on the client computer, period."  So I explained that.  That's something that is very appropriate for an Issue tracker.

Yes it is free software and I appreciate it.  I also take my responsibility seriously to make sure the developers know the use cases the users have.

**Comment by axelchalon** [comment]
> How is it that having Parity installed on my local computer allows me to run the parity signer new-token command on the server?

It doesn't. This is why the code should be rewritten so that Parity UI doesn't download a local instance of Parity Ethereum if it managed to connect to the node; it should simply ask the user to run `parity signer new-token` on the server. Edited https://github.com/parity-js/shell/issues/181#issuecomment-410439729

**Comment by stone212** [comment]
@axelchalon 

> It doesn't. This is why the code should be rewritten so that Parity UI doesn't download a local instance of Parity Ethereum if it managed to connect to the node; it should simply ask the user to run parity signer new-token on the server.

Yes!  Funny, when I said the same thing everyone started yelling at me.
